recipe/conda_build_config.yaml

```diff
@@ -179,18 +185,18 @@ pyqt:
 qt:
   - 5.6
 readline:
-  - 7
+  - 7.0
 r_base:
   - 3.3.2
   - 3.4.1
 scipy:
   - 0.19
 snappy:
-  - 1
+  - 1.1.6
```

Comment:
Historically `snappy` is [not that stable]( https://abi-laboratory.pro/tracker/timeline/snappy/ ). At `1.1.5` symbols were added and removed plus the SONAME changed. Though 1.1.6 and 1.1.7 seem to be ok. Picked the lower end of [what we had before]( https://github.com/conda-forge/conda-forge.github.io/blob/61cdd7fd0374c07e95afdcbd414c7985b141e4b3/scripts/pin_the_slow_way.py#L120 ) again.